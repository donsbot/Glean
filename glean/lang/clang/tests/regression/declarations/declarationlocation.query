query: |
    { N, L } where cxx1.DeclarationLocationName { _, N, L }
transform: [gensort, []]
