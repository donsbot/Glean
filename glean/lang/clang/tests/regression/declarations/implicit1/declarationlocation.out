[
  "@generated",
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 9,
        "columnBegin": 1,
        "lineEnd": 9,
        "columnEnd": 11
      },
      "tuplefield1": "A"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 11,
        "columnBegin": 1,
        "lineEnd": 17,
        "columnEnd": 1
      },
      "tuplefield1": "f"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 12,
        "columnBegin": 3,
        "lineEnd": 12,
        "columnEnd": 5
      },
      "tuplefield1": "a"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 13,
        "columnBegin": 3,
        "lineEnd": 13,
        "columnEnd": 9
      },
      "tuplefield1": "a1"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 14,
        "columnBegin": 3,
        "lineEnd": 14,
        "columnEnd": 14
      },
      "tuplefield1": "a2"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 20,
        "columnBegin": 1,
        "lineEnd": 20,
        "columnEnd": 11
      },
      "tuplefield1": "B"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 22,
        "columnBegin": 1,
        "lineEnd": 37,
        "columnEnd": 1
      },
      "tuplefield1": "g"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 24,
        "columnBegin": 5,
        "lineEnd": 24,
        "columnEnd": 12
      },
      "tuplefield1": "b"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 25,
        "columnBegin": 5,
        "lineEnd": 25,
        "columnEnd": 16
      },
      "tuplefield1": "b1"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 26,
        "columnBegin": 5,
        "lineEnd": 26,
        "columnEnd": 26
      },
      "tuplefield1": "b2"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 31,
        "columnBegin": 5,
        "lineEnd": 31,
        "columnEnd": 14
      },
      "tuplefield1": "b"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 32,
        "columnBegin": 5,
        "lineEnd": 32,
        "columnEnd": 18
      },
      "tuplefield1": "b1"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 33,
        "columnBegin": 5,
        "lineEnd": 33,
        "columnEnd": 30
      },
      "tuplefield1": "b2"
    }
  }
]